test_common/KokkosKernels_TestUtils.hpp

```diff
@@ -93,10 +93,11 @@ struct multivector_layout_adapter<ViewType, false> {
 };
 
 template <class Scalar1, class Scalar2, class Scalar3>
-void EXPECT_NEAR_KK(Scalar1 val1, Scalar2 val2, Scalar3 tol) {
+void EXPECT_NEAR_KK(Scalar1 val1, Scalar2 val2, Scalar3 tol,
```

Comment:
@lucbv: Should this be a macro such that EXPECT_LE reports the correct line numbers upon failure?

Reply:
We can do that, it would be more informative when we have multiple such statements in the same test.